benchmarks/thermal_convection_3d/thermal_convection_3d.prm

```diff
@@ -0,0 +1,172 @@
+# A simple setup for thermal convection in a 3d shell based on 
+# CitcomS benchmark of Arrial et al. 2014. See the manual for more
+# information.
+
+############### Global parameters
+# We use a 3d setup. Here the equations of contuinity, momentum and
+# energy are nondimensionalized with the timescale chosen as the
+# thermal diffusion time of mantle minerals so that time t=1 which
+# corresponds to 265 billion years.
+set Dimension                              = 3
+set Start time                             = 0
+set End time                               = 1
+set CFL number                             = 1
+set Use years in output instead of seconds = false
+set Output directory                       = output-thermal_convection-0
+
+
+############### Parameters describing the model
+# The setup is a 3d spherical shell with a nondimensional length
+# scale chosen as the approximate thickness of the mantle (i.e.
+# mantle thickness = 1).
```

Comment:
This is tricky, originally I used 12 cells along circumference because at low resolution (level 4), the solution is noised at the lower-upper sphere boundary of the standard mesh. 12 cells along the circumference remove this noise. 
After discussion with Cedric, the 12 Cells along circumference would refer to the number of "caps of approximately equal size" that compose the sphere? (Which would then refer to the exact setup of CitcomS - although I use here a lower resolution)
To be honest, I am not sure I really understand the significance of "cells along circumference", I can see one big difference when looking at the mesh (finer mesh in the lower sphere and coarser in the upper sphere), but it does not explain to me what the difference really is: maybe you could try to explain (I find the definition in the manual also vague)?